mail/templates/product_order_receipt/body.html

```diff
@@ -60,6 +60,9 @@ <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px;
                     </table>
                 {% endif %}
                 <strong style="font-weight: 700;">Email Address:</strong> {{ purchaser.email }}<br>
+                {% if purchaser.vat_id %}
```

Comment:
I don't think this is related to the taxes feature as it was an extra thing required to avoid some manual work. @cachob What are your thoughts on this?

Reply:
This is a separate issue from country taxes. A learner may be required to enter a VAT number (by the employer for example) but the country itself may not require a learner's tax